nameko/messaging.py

```diff
@@ -3,22 +3,22 @@
 '''
 from __future__ import absolute_import
 
-import warnings
 from functools import partial
 from logging import getLogger
+import warnings
```

Comment:
i would be -1 on auto-rewriting. status quo is probably fine, though a checking is a fairly fast lint run with e.g isort. also has the bonus of _optionally_ autosorting. something for another day